### PR TITLE
Feature/jwt

### PR DESCRIPTION
**Pull request by 9sanha**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -23,6 +23,7 @@ dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-thymeleaf'
 	implementation 'org.springframework.boot:spring-boot-starter-validation'
 	implementation 'org.springframework.boot:spring-boot-starter-web'
+	implementation 'io.jsonwebtoken:jjwt:0.9.1'
```
Comment (whco) [inline build.gradle:26]: checked

**Review by 9sanha — COMMENTED** [review]
PR TEST

**Review by whco — APPROVED** [review]
checked

**Review by TaegyunWoo — APPROVED** [review]
checked

**Pull request merged**